### ISSUE TITLE
Whether pretraining the LLM in R2GenCSR?

### ISSUE THREAD

**Issue by ZarkPanda** [Event-AHU/Medical_Image_Analysis#21]
I see your paper said that you fine-tuned the LLM:

<img width="526" height="185" alt="Image" src="https://github.com/user-attachments/assets/41a3dbb7-2052-46ab-92df-6093bbc99ee5" />

But in the scripts, the llm_freeze is set to True, and the training methods now will take nearly 80GB already.
So what's the exact meaning of the 'LLM is fine-tuned' in your paper?

**Comment by liyhc** [comment]
Hi! The "fine-tuning" described in the paper refers to optimizing only the report tokens via the instruction prompt (which encodes visual/textual residuals), not full LLM parameter updates. The auto-regressive loss is applied selectively to the report generation part.

**Comment by ZarkPanda** [comment]
Ok I get it, thanks
